src/chrome/content/rules/MayFirstPeopleLink.xml

```diff
@@ -1,9 +1,10 @@
-<!--	See MayFirstPeopleLink-mismatches.xml for problematic rules.
+
```

Comment:
I think this ruleset generated a spurious failure and I reverted it, but some newlines were added in the process.